src/Sluggable/SluggableListener.php

```diff
@@ -27,6 +27,8 @@
  *
  * @author Gediminas Morkevicius <gediminas.morkevicius@gmail.com>
  * @author Klein Florian <florian.klein@free.fr>
+ *
+ * @internal
```

Comment:
should not be internal, as this class is expected to be instantiated by downstream code to use the library